modules/databricks-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/databricks/Databricks.scala

```diff
@@ -42,8 +44,13 @@ object Databricks {
 
           def extendTable(info: ShreddedType.Info): Option[Block] = None
 
-          def getLoadStatements(discovery: DataDiscovery): LoadStatements =
-            NonEmptyList.one(Statement.EventsCopy(discovery.base, discovery.compression, getColumns(discovery)))
+          def getLoadStatements(discovery: DataDiscovery, eventsColumns: List[String]): LoadStatements = {
```

Comment:
Nit: we have now places where we use `columns` but in different contexts, like discovered columns to be loaded or existing columns in DB, so... WDYT about introducing some case classes/aliases for those? E.g. here we could use sth like `Existing/PersistedColumns`?